client/app/components/blueprints/blueprint-editor/blueprint-editor.html

```diff
@@ -0,0 +1,116 @@
+<span ng-class="{'wait-cursor': vm.loading}">
+  <span class="blueprint-designer-header">
+    <ol class="breadcrumb">
+        <li>
+            <a ui-sref="^"> <span class="fa fa-angle-double-left"/>&nbsp;{{'Back to Blueprints'|translate}}</a>
+        </li>
+        <li class="active"><strong translate>Blueprint Editor: </strong>
```

Comment:
Yes, I just want to make sure that we are following what @mzazrivec 's established pattern is.